### PR TITLE
Add Keywords for fetching replicasets in namespace

### PR DESCRIPTION
**Pull request by hello2ray**
\<Remember to add meaningful title\>

\<Short description of the PR\>

Fixes #\<issue number\>

Before merge following needs to be applied:
- [x] At least one example testcase added in testcases/
- [ ] Library Documentation regenerated according to [Generate docs](https://github.com/devopsspiral/KubeLibrary#generate-docs)
- [ ] PR entry added in CHANGELOG.md in **In progress** section
- [ ] All new testcases tagged as **prerelease** along other tags to exclude it from execution until released on PyPI
- [ ] Coverage threshold increased in [.coveragerc](https://github.com/devopsspiral/KubeLibrary/blob/master/.coveragerc) if new coverage is higher than actual, see the lint-and-coverage step in CI
```
fail_under = 67
```

### CODE REVIEW

**Comment by hello2ray** [comment]
Please review

**Comment by m-wcislo** [comment]
Please complete the checklist.

**Comment by hello2ray** [comment]
> Please complete the checklist.
Could you please help me here. What checklist I have to fill here.  Do I need to add any files to generate docs

**Comment by m-wcislo** [comment]
Yeah, so you have the list in the description of the PR. Let me know if you have any problems

File: testcases/replicaset/replicaset.robot [hunk 1]
```diff
@@ -0,0 +1,7 @@
+*** Settings ***
+Resource          ./replicaset_kw.robot
+
+*** Test Cases ***
+Replicaset test case example
+    [Tags]    replicaset
```
Comment (m-wcislo) [inline testcases/replicaset/replicaset.robot:6]: For
> All new testcases tagged as prerelease along other tags to exclude it from execution until released on PyPI
just add **prerelease**

Comment (hello2ray) [reply testcases/replicaset/replicaset.robot:6]: > **prerelease**
I added tags prerelease and could you kindly help me in generate docs and coverage

**Comment by hello2ray** [comment]
Added another one

**Pull request closed without merging**